modules/Bio/EnsEMBL/Transcript.pm

```diff
@@ -2161,9 +2161,21 @@ sub translate {
                                -alphabet => 'dna',
                                -id       => $display_id );
 
-  my $translation =
-    $peptide->translate( undef, undef, undef, $codon_table_id, undef,
-                         undef, $complete5, $complete3 );
+# Signature for BioPerl 1.6 should be (https://metacpan.org/pod/Bio::PrimarySeqI)
+#     my ($terminator, $unknown, $frame, $codonTableId, $complete,
```

Comment:
is it `codontable_id` ?
https://metacpan.org/pod/Bio::PrimarySeqI#translate

Reply:
yes, it is